repotracker/repotracker.go

```diff
@@ -626,7 +626,11 @@ func makeBuildBreakSubscriber(userID string) (*event.Subscriber, error) {
 		if preference == user.PreferenceEmail {
 			subscriber.Target = u.Email()
 		} else if preference == user.PreferenceSlack {
-			subscriber.Target = u.Settings.SlackUsername
+			slackTarget := u.Settings.SlackUsername
```

Comment:
style(?) would it be more readable if we used a switch to check all the different preference options?

Reply:
I agree, but it's a bit out of scope, and I think it's small enough to be worth keeping like this.